src/Components/DraggableResizer/DraggableResizer.tsx

```diff
@@ -126,27 +126,58 @@ export const DraggableResizerRoot: FunctionComponent<DraggableResizerProps> = ({
     }
 
     const newPos = mouseRelativePos / containerSize
-    const newPositions = [...handlePositions]
+    createNewPositions(newPos)
+  }
+
+  const calculateCollapsePosition = (direction: number, dragIndex: number) => {
```

Comment:
Remembering what number corresponds to which direction is going to be hard for someone in the future to debug/improve on it, 

i think we should make an enum here:

```
enum CollapseDirection {
  Left = 0,
  Right = 1,
  Up = 2,
  Down = 3,
}
```

or something similar, that is appropriate. You'd know better what the enum values should be.

Reply:
The values always have to be 0 or 1 though. An enum would not work well here.

Reply:
What is the constraint on it being 0 / 1 only?

Reply:
A collapsed panel will always have a value of 0 or 1. since those refer to the closed left/up and closed right/down. Anything in between is determined by a 0.X value. If the value is 0 or 1, its irrelevent if its up/left or right/down. The code will always collapse it towards the right value.